ftdetect/glsl.vim

```diff
@@ -3,6 +3,6 @@
 
 " Extensions supported by Khronos reference compiler
```

Comment:
Please update this comment - `.glsl` isn't supported by glslang and this can be mentioned here as an exception